### PR TITLE
Update config files of dints with TensorRT

### PR DESCRIPTION
**Pull request by binliunls**
Fixes # .

### Description
Add the TensorRT support for the pancreas_dints_ct_segmentation bundle.

### Status
**Ready/Work in progress/Hold**

### Please ensure all the checkboxes:

- [x] Codeformat tests passed locally by running `./runtests.sh --codeformat`.
- [ ] In-line docstrings updated.
- [ ] Update `version` and `changelog` in `metadata.json` if changing an existing bundle.
- [ ] Please ensure the naming rules in config files meet our requirements (please refer to: `CONTRIBUTING.md`).
- [ ] Ensure versions of packages such as `monai`, `pytorch` and `numpy` are correct in `metadata.json`.
- [ ] Descriptions should be consistent with the content, such as `eval_metrics` of the provided weights and TorchScript modules.
- [ ] Files larger than 25MB are excluded and replaced by providing download links in `large_file.yml`.
- [ ] Avoid using path that contains personal information within config files (such as use `/home/your_name/` for `"bundle_root"`).

### CODE REVIEW

File: models/pancreas_ct_dints_segmentation/docs/README.md
Comment (yiheng-wang-nv) [inline models/pancreas_ct_dints_segmentation/docs/README.md:82]: Thanks for the changes @binliunls , could this bundles be tested by our current ci test function (https://github.com/Project-MONAI/model-zoo/blob/9786cd888a8e3e03cb0e403fb5872a3c3d3e5dd8/ci/verify_tensorrt.py#L69) directly? It seems there are some new args used.
If not, I think we need a more flexible test function, and I will implement it when enhancing our post-merge tests.

Comment (binliunls) [reply models/pancreas_ct_dints_segmentation/docs/README.md:82]: Yes, the ci test function here cannot be performed directly. I updated this part to solve this question. Please take a look at it.

Thanks,
Bin

**Review by yiheng-wang-nv — APPROVED** [review]
Thanks @binliunls , this PR looks good.
I will merge it after #469 is merged (it is helpful to test the new blossom CI pipeline)

**Comment by yiheng-wang-nv** [comment]
suggestion 1 in #422 is also resolved in this PR.

**Pull request merged**
